tests/algo/TestContingencyAlgo.cpp

```diff
@@ -149,7 +149,8 @@ TEST(ContingencyValidation, base) {
   expectedValidContingencies.emplace("hvdcline");
 
   for (auto validContingency : validContingencies.get()) {
-    auto expected = expectedValidContingencies.find(validContingency.get().id);
+    std::cout << "BUBU?! " << validContingency.id << std::endl;
```

Comment:
is this meant to be displayed?

Reply:
haha no I totally forgot this one. Let me remove it

Reply:
solved